commands/deals_daemon_test.go

```diff
@@ -94,8 +98,12 @@ func TestDealsRedeem(t *testing.T) {
 	newWalletBalance, err := minerDaemon.WalletBalance(ctx, minerOwnerAddress)
 	require.NoError(t, err)
 
+	expectedGasCost := 0.1
+	expectedBalanceDiff := float64(dealDuration*dataPriceOneBlock) - expectedGasCost
+	expectedBalanceStr := fmt.Sprintf("%.1f", expectedBalanceDiff)
```

Comment:
Is this rounding still necessary?

Reply:
yes.  I actually had to add it in because otherwise the trailing 0s you get by default from float64 cause this to fail.

Reply:
And note this will still fail if actual balanceDiff is off by a very small fraction because we don't round actualBalanceDiff